IDGPredict/FacetPredict.h

```diff
@@ -91,7 +91,12 @@ class FacetPredict {
   }
 
   void SetMSInfo(std::vector<std::vector<double>>&& bands, size_t nr_stations) {
-    _maxBaseline = 1.0 / std::min(_pixelSizeX, _pixelSizeY);
+    double max_baseline_x = 1.0 / _pixelSizeX;
+    double max_baseline_y = 1.0 / _pixelSizeY;
+    double max_baseline_z = 1.0 / std::min(_pixelSizeX, _pixelSizeY);
+    _maxBaseline = std::sqrt(max_baseline_x * max_baseline_x +
+                             max_baseline_y * max_baseline_y +
```

Comment:
How is this part of adding a test? Should this be here?

Reply:
It is a fix: Without it, idg ignores some baselines while wsclean does not ignore them. Instead of looking at a single dimension for the pixel size, I'm now considering the size of the pixel 'box' in three dimensions for determining _maxBaseline.

Reply:
But where did this code from? Did you make it up yourself, or borrow it from WSClean? If the latter, where?

Reply:
I made it up myself.